src/utils.ts

```diff
@@ -75,11 +102,44 @@ export const DOTFILE = {
     const projectDirectory: string = findParentDir.sync(process.cwd(), DOT.PROJECT.PATH) || DOT.PROJECT.DIR;
     return dotf(projectDirectory, DOT.PROJECT.NAME);
   },
-  // See `login`: Stores { accessToken, refreshToken }
+  // Stores {ClaspSettingsDefault}
   RC: dotf(DOT.RC.DIR, DOT.RC.NAME),
+  // Stores {ClaspSettingsLocal}
   RC_LOCAL: dotf(DOT.RC.LOCAL_DIR, DOT.RC.NAME),
 };
 
+/**
+ * Checks if local OAuth client settings rc file exisits.
+ * @return {boolean}
+ */
+export const localOathSettingsExist = (): boolean =>
```

Comment:
Combine `localOathSettingsExist` and `defaultOathSettingsExist`. Something like:
```ts
export function oAuthSettingsExist(local: boolean):boolean {
  return fs.existsSync(local ? DOT.RC.ABSOLUTE_LOCAL_PATH : DOT.RC.ABSOLUTE_PATH);
}
```

Reply:
Combining into single export and using `isLocalCreds` to differentiate saves only 1-2 lines in `utils.ts`. Will have a look and refactor.